hooks/shapediver/viewer/interaction/useNodeInteractionData.ts

```diff
@@ -79,7 +50,7 @@ export function useNodeInteractionData(sessionId: string, outputIdOrName: string
 		 * @param node 
 		 * @param interactionDataSettings 
 		 */
-		const addInteractionData = (node: ITreeNode, interactionDataSettings: { select?: boolean, hover?: boolean, drag?: boolean}) => {
+		const addInteractionData = (node: ITreeNode, interactionDataSettings: { select?: boolean, hover?: boolean, drag?: boolean }) => {
```

Comment:
@MajorMeerkatThe3rd this looks like a helper function for managing data in `nodesWithInteractionData`, seems like we can move it outside of here to the global scope of this file (where `nodesWithInteractionData` is defined).